src/main/kotlin/app/revanced/patches/youtube/layout/autocaptions/fingerprints/StartVideoInformerFingerprint.kt

```diff
@@ -0,0 +1,31 @@
+package app.revanced.patches.youtube.layout.autocaptions.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.annotation.MatchingMethod
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patches.youtube.layout.autocaptions.annotations.AutoCaptionsCompatibility
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+@Name("startvideo-informer-fingerprint")
+@MatchingMethod("Laaxf;", "d")
+@AutoCaptionsCompatibility
+@Version("0.0.1")
+object StartVideoInformerFingerprint : MethodFingerprint(
+    "V", AccessFlags.PUBLIC or AccessFlags.FINAL, listOf("L", "L", "L", "L"), listOf(
+        Opcode.INVOKE_STATIC,
+        Opcode.IGET_OBJECT,
+        Opcode.IGET_OBJECT,
+        Opcode.NEW_INSTANCE,
+        Opcode.INVOKE_DIRECT,
+        Opcode.INVOKE_INTERFACE,
+        Opcode.IF_EQZ,
+        Opcode.CONST_STRING,
+        Opcode.INVOKE_INTERFACE,
+        Opcode.IGET_OBJECT,
+        Opcode.INVOKE_VIRTUAL,
+        Opcode.MOVE_RESULT,
```

Comment:
If it is possible, create a more durable fingerprint. This one could fail if the parameters or the opcode pattern do not match.

Reply:
Added a fuzzy 3, to strengthen fingerprint.